Animations/Animation.js

```diff
@@ -107,14 +108,14 @@ var makeTransition = function (displayName, transitions) {
       return (
         React.DOM.div(
           {
-            className: displayName,
+            className: className,
             key: this.props.key
```

Comment:
You can't access the key anymore in the latest version of React. If the key needs to be accessed, I'd recommend adding an additional prop of 'id'.